tutorials/plot_morph_stc.py

```diff
@@ -0,0 +1,248 @@
+# -*- coding: utf-8 -*-
+r"""
+===========================================
+Morphing source estimates using SourceMorph
+===========================================
```

Comment:
The new tutorial reads great. It's simple and easy to follow! But see my comment about last section